frontend/appflowy_flutter/test/bloc_test/board_test/create_card_test.dart

```diff
@@ -20,16 +20,16 @@ void main() {
     )..add(const BoardEvent.initial());
     await boardResponseFuture();
 
-    final groupId = boardBloc.state.groupIds.first;
+    final groupId = boardBloc.state.groupIds.last;
```

Comment:
Instead of a fixed index, I feel it more appropriate to take the group which `isDefault`, since that will work no matter if you change the ordering. What do you think?

Reply:
Good point, though I think part of the test is to make sure that that stack is the last group as well